src/librustc_mir/interpret/eval_context.rs

```diff
@@ -280,13 +280,13 @@ impl<'a, 'tcx, M: Machine<'tcx>> EvalContext<'a, 'tcx, M> {
     pub fn size_and_align_of_dst(
         &mut self,
         ty: ty::Ty<'tcx>,
-        value: Value,
+        value: Value, // This has to be a fat ptr; we only care about the "extra" data in it.
```

Comment:
That comment isn't quite true. In case the type is `Sized`, we don't care about the value at all.

Also: move the comment to a doc comment on the function.